### DISCUSSION TITLE
[2021.08.26] null 처리하기

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#59] [SQL]
[null 처리하기](https://programmers.co.kr/learn/courses/30/lessons/59410)

**Comment by pyunji** [comment]
[null 처리하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59410.sql)

**Comment by hyk339** [comment]
[null 처리하기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/7d44a7fc32ea3487324b4aa3630637b724d76e46)

**Comment by leechang4** [comment]
[null 처리하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059410.sql)

**Comment by chaeyeon4u** [comment]
[null 처리하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59410.sql)
